package.json

```diff
@@ -27,11 +27,12 @@
   ],
   "scripts": {
     "lint": "jshint bin lib test",
-    "mocha": "_mocha --reporter spec test/svg-sprite.js",
+    "mocha": "_mocha --reporter spec test/**/*.spec.js test/svg-sprite.js",
     "coverage": "c8 npm run mocha",
     "test": "npm run mocha"
   },
   "dependencies": {
+    "@ert78gb/async-init": "^1.0.0",
```

Comment:
Not sure why this is needed. Personally I don't want to add a new dep regardless its size.

Reply:
It is responsible for the async singleton creation, the puppeteer browser creation is an async operation. It handles correctly if more than 1 request arriving before the browser created.
If you don't want to add it as a dependency then I have to add files (implementation and test) to this package.

I thought on to create a standalone package for the `BrowserManager` too because it is also reusable.
But in this case, I will not do.

Reply:
I copied the code from the package to this repo, but not logical for me. I always create small packages if the code useful in other project too. Now we increased the code base and test cases of this package.